doc/API.md

```diff
@@ -198,38 +341,38 @@ struct {
 	upcomingrevenue   types.Currency (string)
 }
 ```
-`collateral` is the number of hastings per byte per block that are put up as
+'collateral' is the number of hastings per byte per block that are put up as
 collateral when making file contracts.
 
-`ipaddress` is the network address of the host.
+'ipaddress' is the network address of the host.
```

Comment:
it should be 'netaddress'

'address' is confusing because 'address' means cryptographic address or wallet address depending on context.